outofcore/include/pcl/outofcore/impl/octree_base_node.hpp

```diff
@@ -1639,7 +1639,8 @@ namespace pcl
               random_sampler.setSample (static_cast<unsigned int> (sample_points));
 
               pcl::ExtractIndices<pcl::PCLPointCloud2> extractor;
-              
+			  extractor.setInputCloud(tmp_blob);
```

Comment:
Please replace tabs with spaces. Also add a space between function name and brackets to match PCL style. Thanks!

Reply:
Please check. I hope everything is ok now. Thanks.